src/patternfly/layouts/Split/examples/Split.md

```diff
@@ -87,13 +87,13 @@ import './Split.css'
 
 ## Documentation
 ### Overview
-The split layout is designed to position items horizontally, with one item filling the available horizontal space.
+The split layout is designed to position items horizontally.
 
 ### Usage
 | Class | Applied to | Outcome |
 | -- | -- | -- |
-| `.pf-v6-l-split` | `<div>`, `<section>`, or `<article>` | Initiates the split layout. |
-| `.pf-v6-l-split__item` | `<div>` | Initiates a split item. **Required** |
+| `.pf-v6-l-split` | `*` | Initiates the split layout. |
+| `.pf-v6-l-split__item` | `*` | Initiates a split item. |
```

Comment:
This is not required - prior to this PR, it was only required for `.pf-m-fill`, but with this PR it isn't required at all.